sdk/eventhub/eventhubs-checkpointstore-tables/test/tables-checkpointstorespec.ts

```diff
@@ -0,0 +1,11 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { expect } from "chai";
+import { TableCheckpointStore } from "../src";
+
+describe("Table Checkpoint Store", function(): void {
+  it("implements functions for table checkpoint store", () => {
+    expect(TableCheckpointStore, "to implement checkpoint");
```

Comment:
I'm not sure what this test is checking. I think for now, having a test that ensures that `TableCheckpointStore` exists is good enough.

(That can be as simple as asserting that `typeof TableCheckpointStore` is `function`.)